poc-data-api/useful_spatial_functions_data_extraction.R

```diff
@@ -0,0 +1,145 @@
+# Useful functions to extract data using polygons

```

Comment:
maybe documenting each function using `roxygen2`. But this is not a priority right now.